### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by oafischer**

### CODE REVIEW

**Review by cadawson — APPROVED** [review]
Links well. Thanks!

File: apis/channels.json.adoc [hunk 1]
```diff
@@ -276,7 +276,7 @@ __optional__|API version of the referent.|string
 |**fieldPath** +
 __optional__|If referring to a piece of an object instead of an entire object, this string should contain a valid JSON/Go field access statement, such as desiredState.manifest.containers[2]. For example, if the object reference is to a container within a pod, this would take on a value like: "spec.containers{name}" (where "name" refers to the name of the container that triggered the event) or if no container name is specified "spec.containers[2]" (container with index 2 in this pod). This syntax is chosen only to have some well-defined way of referencing a part of an object. TODO: this design is not final and this field is subject to change in the future.|string
 |**kind** +
-__optional__|Kind of the referent. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds|string
+__optional__|Kind of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/
```
Comment (dockerymick) [inline apis/channels.json.adoc:279]: I recommend the following:

For more information, see link:https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/

External link guidance: https://stolostron.github.io/rhacm-docs/wr_linking

**Review by dockerymick — APPROVED** [review]
Made a suggestion. Thanks for the update!

File: apis/channels.json.adoc (hunk 1 above)
Comment (oafischer) [reply apis/channels.json.adoc:279]: Thanks, I'm going to leave it as is for now for consistency but let's definitely address all similar instances in a future issue or similar.

**Pull request merged**